opal/urls.py

```diff
@@ -67,24 +65,10 @@
     url(r'api/v0.1/', include(api.router.urls)),
     url(r'^templates/record/(?P<model>[a-z_\-]+).html$',
         views.RecordTemplateView.as_view(), name="record_view"),
+    url(r'^templates/modals/(?P<model>[a-z_\-]+).html/(?P<list>[a-z_\-]+/?)$',
+        views.ModalTemplateView.as_view(), name="modal_view"),
 )
 
-# Generated subrecord template views
-for subrecord_model in subrecords():
-    sub_url = camelcase_to_underscore(subrecord_model.__name__)
-    urlpatterns += patterns(
-        '',
-        url(r'^templates/forms/%s.html/?$' % sub_url,
-            views.FormTemplateView.as_view(), {'model': subrecord_model},
-            name="form_template_view"
-            ),
-        url(r'^templates/modals/%s.html/?$' % sub_url,
```

Comment:
What's happened to /templates/forms/foo ? Do we not use/need it? If so, do we still need the view ?